src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -363,6 +363,9 @@ public static Config BaseConfig
                       @"akka {
                         loglevel = ""WARNING""
                         stdout-loglevel = ""WARNING""
+                        coordinated-shutdown.terminate-actor-system = off
```

Comment:
Needed to disable `CoordinatedShutdown` from running here, so nodes can still pass barriers in the even that they leave the cluster